Problem2.java

```diff
@@ -0,0 +1,37 @@
+// Time Complexity - O(n^2) where n is the number of elements in the array
+//Space Complexity - O(1)
```

Comment:
How is time complexity O(n^2)

Reply:
For each i th element in the array,  it is traversing from i+1 till the end to get unique possible triplets.

Reply:
You are traversing the elements once. Recall the algorithm.

Reply:
The complexity given by you is the correct one.